contracts/Makefile

```
@@ -0,0 +1,20 @@
+.PHONY: all deps compile
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)